src/main.py

```diff
@@ -89,7 +90,7 @@ def get_dataset_by_id(
     request: Request,
     response: Response,
     dataset_id: str,
-    metadata_store: StubMetadataStore = Depends(StubMetadataStore),
+    metadata_store: OxigraphMetadataStore = Depends(OxigraphMetadataStore),
```

Comment:
I think this needs to be changed back to StubMetadataStore until all of the Oxigraph methods are done (also in test_dataset.py).